### PR TITLE
Adding CTA

### PR DESCRIPTION
**Pull request by Azahires**
Search component + LocaliseMe button without connecting API.

### CODE REVIEW

**Review by cartige — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-icons": "^4.6.0"
   }
```
Comment (LoicBrassart) [inline package.json]: Not here ; in the `frontend` one !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Almost there, please fix ;-)

**Comment by LoicBrassart** [comment]
Also: NOPE !
You do you PRs towards dev, not main ! ⚠️

**Review by CharlyJuteau — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Conflicts + Missing SM review: I'll come back later, you seem occupied :p

**Review by CharlyJuteau — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**